### DISCUSSION TITLE
图谱驱动的大语言模型 Llama Index - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#34] [Comments]
# 图谱驱动的大语言模型 Llama Index - siwei.io

如何利用图谱构建更好的 In-context Learning 大语言模型应用。

https://siwei.io/graph-enabled-llama-index/

**Comment by robbin2012** [comment]
老师，请教这个知识图谱是用怎么生成的？谢谢！
